main.go

```diff
@@ -20,9 +20,13 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/prometheus/common/version"
-	"k8s.io/klog/v2"
+	"gopkg.in/yaml.v3"
+	"k8s.io/klog"
```

Comment:
Is the switch to klog/v1 here intentional?

Reply:
nope, fixed